packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -304,17 +308,26 @@ contract DeployImplementations is Script {
     )
         internal
         virtual
-        returns (OPStackManager opsm_)
+        returns (OPStackManager opsmProxy_)
     {
         SuperchainConfig superchainConfigProxy = _dii.superchainConfigProxy();
         ProtocolVersions protocolVersionsProxy = _dii.protocolVersionsProxy();
 
+        // In this case, DeployImplementations script needs to deploy a proxy contract and the opsm contract.
+        // To deploy a proxy contract, you need to know the proxy admin.
+        // However, this proxy admin is not deployed until the OPStackManager.
+        // Not using blueprint because we don't have access to l2ChainId.
         vm.broadcast(msg.sender);
-        opsm_ = new OPStackManager({
+        // Setting proxy admin to msg.sender because ProxyAdmin is not deployed yet.
+        Proxy proxy = new Proxy(msg.sender);
```

Comment:
I'd say we use the `superchainProxyAdmin` address that's output from the `DeploySuperchain` script as the owner of this proxy